src/data/randomizer/special_form_tables.h

```diff
@@ -0,0 +1,74 @@
+#include "global.h"
+#include "constants/species.h"
+
+const u16 sPaldeanTaurosFormChoices[] = {
```

Comment:
any reason not to use the already present form tables in expansion?

Reply:
I think I have that explained somewhere.
For those cases, it would not work, but for different reasons:
* Floette would allow Eternal Flower Floette to appear.
* Tauros would cause Kanto Tauros to show up in places where Paldea Tauros should sometimes and thus also be more likely that it should be.
* Minior would allow core forms to appear in the wild, which should not happen.